Server/Forms/FrmMain.cs

```diff
@@ -248,54 +251,58 @@ private void ClientRead(Server server, Client client, IPacket packet)
             {
                 CommandHandler.HandleStatus(client, (Core.Packets.ClientPackets.Status) packet);
             }
-            else if (type == typeof (Core.Packets.ClientPackets.UserStatus))
+            else if (type == typeof(Core.Packets.ClientPackets.RecoverPassResponse))
```

Comment:
You should put these two a bit lower down this decision tree to allow better speed for the more performance-critical commands (such as remote desktop data processing or handling file downloads).